src/wallets/sendTransaction/SendAdvancedTransaction.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {FunctionComponent} from 'react'
+import {ModalProps} from 'antd/lib/modal'
+import {ModalLocker} from '../../common/MantisModal'
+import {Dialog} from '../../common/Dialog'
+import {DialogInput, DialogTextArea} from '../../common/dialog/DialogInput'
+import {DialogColumns} from '../../common/dialog/DialogColumns'
+import {Wei} from '../../common/units'
+import {DialogAddressSelect} from '../../address-book/DialogAddressSelect'
+import {FeeEstimates} from '../../common/wallet-state'
+import {useTranslation} from '../../settings-state'
+import {AdvancedTransactionParams} from './common'
+
+interface SendAdvancedTransactionProps {
+  onSend: () => void
+  availableAmount: Wei
+  transactionParams: AdvancedTransactionParams
+  setTransactionParams: (advancedParams: Partial<AdvancedTransactionParams>) => void
+  estimateTransactionFee: () => Promise<FeeEstimates>
+  onCancel: () => void
+}
+
+export const SendAdvancedTransaction: FunctionComponent<SendAdvancedTransactionProps &
+  ModalProps> = ({
+  transactionParams,
+  setTransactionParams,
+  onCancel,
+  onSend,
+}: SendAdvancedTransactionProps & ModalProps) => {
+  const {t} = useTranslation()
+  const modalLocker = ModalLocker.useContainer()
+
+  const {amount, recipient, gasLimit, gasPrice, data, nonce} = transactionParams
+
+  return (
+    <>
+      <Dialog
+        title={t(['wallet', 'title', 'send'])}
+        leftButtonProps={{
+          onClick: onCancel,
+          disabled: modalLocker.isLocked,
+        }}
+        rightButtonProps={{
+          children: t(['wallet', 'button', 'sendTransaction']),
+          onClick: onSend,
+        }}
+        onSetLoading={modalLocker.setLocked}
+        type="dark"
+      >
+        <DialogAddressSelect
```

Comment:
For "advanced" it should be either empty or a valid address.
I see that none of the validations work here, I won't add separate comments then.